h2o-core/src/main/java/water/fvec/NewChunk.java

```diff
@@ -239,6 +264,33 @@ public void resize(int len) {
   private transient double _ds[];   // Doubles, for inflating via doubles
   public transient byte[]   _ss;   // Bytes of appended strings, including trailing 0
   private transient int    _is[];   // _is[] index of strings - holds offsets into _ss[]. _is[i] == -1 means NA/sparse
+    
+  public long byteSize() {
```

Comment:
if we have some categoricals stored as strings.. will the logic to detect that string size be somewhere here too? 
empty strings in jvm for example use up 40 bytes. 
https://www.javaworld.com/article/2077496/java-tip-130--do-you-know-your-data-size-.html

Reply:
string data is stored in _ss and _is and added to the sum

Reply:
great. thanks for confirming @honzasterba